addon/components/mobile-menu/tray.js

```diff
@@ -13,6 +14,11 @@ import { disableBodyScroll, enableBodyScroll } from 'body-scroll-lock';
  * @hide
  */
 export default class TrayComponent extends Component {
+  fastboot = getOwner(this).lookup('service:fastboot');
+  get isFastBoot() {
+    return !!this.fastboot?.isFastBoot;
+  }
```

Comment:
```suggestion
  get fastboot() {
    return getOwner(this).lookup('service:fastboot');
  }
  
  get isFastBoot() {
    return !!this.fastboot?.isFastBoot;
  }
```

To be in line with the updated syntax on master.